authority/voting/client/client.go

```diff
@@ -180,7 +180,11 @@ func (p *connector) initSession(ctx context.Context, doneCh <-chan interface{},
 		AuthenticationKey:  linkKey,
 		RandomReader:       rand.Reader,
 	}
-	s, err := wire.NewPKISession(cfg, true)
+
+	// was previously s, err := wire.NewPKISession(cfg, true)
+	// where true indicates initiator
+	// see ticket https://github.com/katzenpost/katzenpost/issues/630
```

Comment:
there's only a few call sites in which to make this perfectly clear what has happened. if we switch from pqXX to a different pattern then this will be revisited.